core/src/main/java/org/geysermc/geyser/network/netty/GeyserServer.java

```diff
@@ -84,7 +84,7 @@ public final class GeyserServer {
     /*
     The following constants are all used to ensure the ping does not reach a length where it is unparsable by the Bedrock client
      */
-    private static final String PING_VERSION = pingVersion();
+    private static final String PING_VERSION = GameProtocol.DEFAULT_BEDROCK_CODEC.getMinecraftVersion();
```

Comment:
I'm not against this, but one thing that should be noted here is that because we no longer override the minecraftVersion the ping version will now become 1.21.20 instead of 1.21.21 (which was the version that'd show in the ping before this PR)

Reply:
good catch